### ISSUE TITLE
Hide Mini-Cart drawer blocks from global styles UI

### ISSUE THREAD

**Issue by Aljullu** [woothemes/woocommerce#42369]
Blocks used in the Mini-Cart template part are visible in the global styles UI, even though there is only one instance of them in the entire site. Because of that, we should hide them from the global styles UI.

## To reproduce

1. Go to Appearance > Editor > Styles > Blocks and search for `Mini-Cart`.
2. Notice all blocks rendered inside the Mini-Cart template part are there:
![imatge](https://github.com/woocommerce/woocommerce-blocks/assets/3616980/787bd6a9-581c-4364-88ee-ff1e9c088969)

## Expected behavior

We should only leave the `Mini-Cart` block, but hide all the other ones.

**Comment by tjcafferkey** [comment]
This issue _might_ be blocked by this https://github.com/WordPress/gutenberg/issues/39201

**Comment by Aljullu** [comment]
Oooh, thanks for connecting the dots, @tjcafferkey! I completely forgot about that issue!

**Comment by tjcafferkey** [comment]
No problem @Aljullu. It might be worth commenting on this issue with our use case to help provide that additional context.

**Comment by Aljullu** [comment]
Isn't that already @gigitux's comment in that PR? We have some more blocks affected by this since the issue was created, but the use-case is still the same. :thinking:

**Comment by tjcafferkey** [comment]
Yes true, maybe we should reword it to present it in a more unbiased way, highlighting the benefits for other theme/plugin developers but then using our use-case as a supporting argument.

**Comment by Aljullu** [comment]
I'm reopening because we still would like to fix this at some point. There are some blocks that would also benefit from this, like *Product Category Title*.

**Comment by kkoteen** [comment]
Hi @Aljullu - thanks for reporting this, and for the detail you gave.

I know this has been open for a very long time, and I appreciate your patience. We're currently going through some older WooCommerce issues, identifying what still needs fixing, and tidying up what's already fixed or out of scope.

It looks like this has already been fixed, or it's no longer happening on trunk.

Here's what I found: Mini-Cart inner blocks on trunk still expose block supports in their block metadata, so the global styles UI can surface them. Issue discussion explicitly ties the desired fix to a Gutenberg limitation, and no local core-side workaround is evident in current block metadata or editor code.

I'm going to close this one. If you're still facing this issue on a current version, please reply with steps to reproduce and we'll happily take another look.
